### PR TITLE
Add browserstack credentials support

### PR DESCRIPTION
**Pull request by vStone**
Add support for browserstack credentials

### CODE REVIEW

File: spec/unit/puppet/provider/jenkins_credentials/cli_spec.rb [hunk 1]
```diff
@@ -360,6 +368,40 @@
     end
   end
 
+  shared_examples 'a provicer for example hash 9' do
```
Comment (jhoblitt) [inline spec/unit/puppet/provider/jenkins_credentials/cli_spec.rb]: typo

**Comment by jhoblitt** [comment]
I've kicked travis... one job failed because `bundler` wasn't installed...

**Comment by vStone** [comment]
@jhoblitt quick sign-off on this one? I would rather not keep rebasing :)

**Comment by bastelfreak** [comment]
Thanks for the PR @vStone !

**Pull request merged**